### PR TITLE
Dont load jQuery file if JQUERY_URL is set to empty string

### PR DESCRIPTION
**Pull request by hebertjulio**

### CODE REVIEW

**Comment by SalahAdDin** [comment]
I think that the jquery version should be updated, with 1.3 versio i have conflicts.

File: smart_selects/widgets.py [hunk 1]
```diff
@@ -46,12 +46,12 @@ def __init__(self, app_name, model_name, chain_field, model_field,
...
 
     class Media:
-        extra = '' if settings.DEBUG else '.min'
-        js = [
-            'jquery%s.js' % extra,
-            'jquery.init.js'
-        ]
         if USE_DJANGO_JQUERY:
+            extra = '' if settings.DEBUG else '.min'
+            js = [
```
Comment (blag) [inline smart_selects/widgets.py]: Doesn't `js` immediately get overwritten by [line 55](https://github.com/digi604/django-smart-selects/pull/87/files#diff-7bd64618692ef6f50012e6d8dbc251e7L55)?

File: README.md [hunk 2]
```diff
@@ -13,25 +13,25 @@ If you have the following model:
 		area = models.ForeignKey(Area)
 		city = models.CharField(max_length=50)
 		street = models.CharField(max_length=100)
-
```
Comment (blag) [inline README.md]: Please put whitespace changes into their own commit.

**Comment by blag** [comment]
Superceded by f474f47874cda15fb2a58a90473530b81ab73ce5. Closing.
